new-lamassu-admin/src/components/editableTable/Row.js

```diff
@@ -1,250 +1,150 @@
 import { makeStyles } from '@material-ui/core'
+import classnames from 'classnames'
+import { Field, useFormikContext } from 'formik'
+import React from 'react'
 
-import { Link } from 'src/components/buttons'
-import { Td, Tr, CellDoubleLevel } from 'src/components/fake-table/Table'
-import { TextInputDisplay } from 'src/components/inputs/base/TextInput'
-import { ReactComponent as DeleteIcon } from 'src/styling/icons/action/delete/enabled.svg'
+import { Link, IconButton } from 'src/components/buttons'
+import { Td, Tr } from 'src/components/fake-table/Table'
+import { TL2 } from 'src/components/typography'
 import { ReactComponent as DisabledDeleteIcon } from 'src/styling/icons/action/delete/disabled.svg'
-// import { ReactComponent as EditIcon } from 'src/styling/icons/action/edit/enabled.svg'
-// import { ReactComponent as DisabledEditIcon } from 'src/styling/icons/action/edit/disabled.svg'
+import { ReactComponent as DeleteIcon } from 'src/styling/icons/action/delete/enabled.svg'
+import { ReactComponent as DisabledEditIcon } from 'src/styling/icons/action/edit/disabled.svg'
+import { ReactComponent as EditIcon } from 'src/styling/icons/action/edit/enabled.svg'
 
-const styles = {
-  button: {
-    border: 'none',
-    backgroundColor: 'transparent',
-    outline: 0,
-    cursor: 'pointer'
-  },
-  actionCol: {
-    display: 'flex',
-    marginLeft: 'auto'
-  },
-  actionColDisplayMode: {
-    justifyContent: 'center'
-  },
-  actionColEditMode: {
-    justifyContent: 'flex-end',
-    '& > :first-child': {
-      marginRight: 16
-    }
-  },
-  textInput: {
-    '& > .MuiInputBase-input': {
-      width: 282
-    }
-  }
-  // doubleLevelRow: {
-  //   '& > div': {
-  //     marginRight: 72
-  //   }
-  // }
-}
+import styles from './Row.styles'
+import { ACTION_COL_SIZE } from './consts'
 
 const useStyles = makeStyles(styles)
 
-const ERow = memo(
-  ({ elements, editing, setEditing, disableAction, action }) => {
-    const classes = useStyles()
+const ActionCol = ({
+  editing,
+  setEditing,
+  enableEdit,
+  disabled,
+  onDelete,
+  enableDelete
+}) => {
+  const classes = useStyles()
+  const { values, submitForm, resetForm } = useFormikContext()
+
+  const actionColSize =
+    enableDelete && enableEdit ? ACTION_COL_SIZE / 2 : ACTION_COL_SIZE
 
-    const Cell = ({
-      name,
-      input,
-      type,
-      display,
-      className,
-      size,
-      textAlign,
-      inputProps,
-      editing
-    }) => {
-      return (
-        <Td size={size} textAlign={textAlign}>
-          {editing && (
-            <Field
-              id={name}
-              name={name}
-              component={input}
-              className={className}
-              {...inputProps}
-            />
-          )}
-          {!editing && type === 'text' && (
-            <TextInputDisplay display={display} {...inputProps} />
-          )}
+  return (
+    <>
+      {editing && (
+        <Td textAlign="center" width={ACTION_COL_SIZE}>
```

Comment:
Hey, this was a bit confusing at this version. The reason why it was left with the constant is because when editing all actions icons are replaced with 'cancel' and 'save' buttons. So this is always the same size regardless of how many actions exist.